projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -360,9 +231,89 @@ protected string GetDateOfDelivery()
         /// <param name="value"></param>
         protected void SetDateOfDelivery(string value)
         {
-            string time = this.GetBirthTime();
-            this.Subject.BirthDateElement = ConvertToDate(value);
-            this.SetBirthTime(time);
+            if (String.IsNullOrEmpty(value))
```

Comment:
do you think it would make more sense to throw an argument exception here since it is required and shouldn't be set to null?

Reply:
Actually, I guess that's not true for the mother or father birth date right? only the child or fetus? so those checks would need to happen earlier in the specific setters. If it does reach this point, for example if father birth date was set to null, I think it should remove the birth date.

Reply:
Updated to remove date data when set to null/blank